app/models/review.rb

```diff
@@ -0,0 +1,9 @@
+class Review < ApplicationRecord
+  belongs_to :shelter
+
+  validates_presence_of :title
+  validates_presence_of :rating
+  validates_presence_of :content
+  validates_presence_of :shelter
+  validates_presence_of :picture
```

Comment:
Do we have a model test for Review?

Reply:
Good call. No we do not. I completely spaced when trying to get this through but I will go ahead and make one.